testapi.php

```diff
@@ -0,0 +1,117 @@
+<?php
+class MagentoAPI
+{
+    protected $_magentoBaseUrl = 'https://kalexey.local/';
```

Comment:
Didn't work for me with https. While changing to http it began working. Seems some additional configurations are needed with curl to perform https requests. No need to fix, just keep it in mind

Reply:
thanks